aider/main.py

```diff
@@ -380,6 +380,12 @@ def main(argv=None, input=None, output=None, force_git_root=None):
         metavar="COMMAND",
         help="Specify a single message to send GPT, process reply then exit (disables chat mode)",
     )
+    other_group.add_argument(
+        "--message-file",
+        "-mf",
```

Comment:
It's unusual to have a single `-` followed by 2 letters. I'd be fine with a short form like `-f`.

Reply:
Absolutely! Done!